tasktiger/migrations.py

```diff
@@ -0,0 +1,20 @@
+def migrate_executions_count(tiger):
+    """
+    Backfills ``t:task:<uuid>:executions_count`` by counting
+    elements in ``t:task:<uuid>:executions``.
+    """
+
+    migrate_task = tiger.connection.register_script(
+        """
+        local count = redis.call('llen', KEYS[1])
+        if tonumber(redis.call('get', KEYS[2]) or 0) < count then
+            redis.call('set', KEYS[2], count)
+        end
+        """
+    )
+
+    for key in tiger.connection.scan_iter(
+        count=100, match="*:task:*:executions"
```

Comment:
Should the `match` just include the `REDIS_PREFIX` alreaady and then we don't have to do the check below?

Reply:
8405c95314c94cf6f95e4ed2eecd25d402e0b01a